### DISCUSSION TITLE
Tasmota sends TAB in json string which is invalid

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#12672] [Q&A] [answered]
In the URL of the OTA is for unknown reason a tab character included which should be an encoded tab (\t) instead. This causes an json parsing error in Nodered.

https://github.com/node-red/node-red/issues/3071

Is this a bug or can I solve this myself?

Tasmota 9.4.0.4

**Comment by sfromis** [comment, marked as answer]
Since the OtaUrl string there is not the original one from Tasmota, you may accidentally have set it including the tab character.

**Reply by jeroenst** [reply]
> Since the OtaUrl string there is not the original one from Tasmota, you may accidentally have set it including the tab character.

Can I change the url somewhere?

**Reply by sfromis** [reply]
Sure. You have the command OtaUrl. Or if you do an upgrade via an URL.

**Comment by Jason2866** [comment]
If this is a bug in v.9.4 it will not be fixed. Please use latest version v.9.5. Bug fixes are done only for latest version.

**Comment by jeroenst** [comment]
Ok, removed the tab in the otaurl, so the problem for me has gone.

Still Tasmota ideally shouldn't send out a tab character in a json string but a \t instead.

**Reply by barbudor** [reply]
Tasmota just send whatever it is requested to.
There shouldn't be neither a tab 0x09 or a "\t" in a URL.
First it doesn't make sense
Then if for any crazy reason, you really want to have non printable characters in a URL you must have it URL-encoded like "%09".
